client/components/SellButton.jsx

```diff
@@ -1,19 +1,37 @@
 import React from 'react';
 import Button from '@mui/material/Button';
 
-function SellButton({ user, ticker, sellQty }) {
+function SellButton({ user, ticker, sellQty, setUserPort, userPort }) {
+  const userPortAfterSell = (sym, quant) => {
+    const newPortValues = [];
+    for (let i = 0; i < userPort.length; i++) {
+      if (userPort[i].symbol === sym) {
+        const soldTickerVals = { ...userPort[i] };
+        if (soldTickerVals.qty - Number(quant) <= 0) {
```

Comment:
If the quantity is more than I have to sell, I still want to add this userPort[i] to my newPortValues.  However, I just won't be changing the quantity.   if I have 5 shares of AAPL and I want to sell 10, then I'll just be left with my 5 shares.